docs/README.md

```diff
@@ -52,67 +52,93 @@ You'll need a `kubectl`-style config file to connect to your cluster.
    plugins. It can be any directory that is visible to the user.
 
-2. `lookupPluginsInPath` which is the same as the persistent flag
-   `--lookup-plugins-in-path` and specifies if `kn` should look for plugins
-   anywhere in the specified `PATH` environment variable. This is a boolean
-   configuration option and the default value is `false`.
-
-3. `sink` defines your prefix to refer to Kubernetes addressable resources. To
-   configure a sink prefix, define following in the config file:
-   1. `prefix`: Prefix you want to describe your sink as. `service` or `svc`
-      (`serving.knative.dev/v1`) and `broker` (`eventing.knative.dev/v1alpha1`)
-      are predefined prefixes in `kn`. These predefined prefixes can be
-      overridden by values in configuration file.
-   2. `group`: The APIGroup of Kubernetes resource.
-   3. `version`: The version of Kubernetes resources.
-   4. `resource`: The plural name of Kubernetes resources (for example:
-      services).
-
-For example, the following `kn` config will look for `kn` plugins in the user's
-`PATH` and also execute plugin in `~/kn/.config/plugins`. It also defines a sink
-prefix `myprefix` which refers to `brokers` in `eventing.knative.dev/v1alpha1`.
-With this configuration, you can use `myprefix:default` to describe a Broker
-`default` in `kn` command line.
+* `path-lookup`, which is the same as the persistent flag
+   `--lookup-plugins-in-path` and specifies if `kn` should look for plugins anywhere in the specified `PATH` environment variable. This option is a boolean type, and the default value is `false`.
 
-```bash
-cat ~/.config/kn/config.yaml
-lookupPluginsInPath: true
-pluginsdir: ~/.config/kn/plugins
-sink:
-- prefix: myprefix
-  group: eventing.knative.dev
-  version: v1alpha1
-  resource: brokers
-```
+#### Eventing configuration
+
+The `eventing:` top-level configuration section holds configuration options that influence the behaviour of eventing related commands.
 
----
+* `sink-mappings` defines prefixes to refer to Kubernetes _Addressable_ resources as
+   used in `--sink` kind of options. `sink-mappings refers to an array of individual mapping definitions that you can configure with these fields:
+   - `prefix`: Prefix you want to describe your sink as. `ksvc`
+      (`serving.knative.dev/v1`), `broker` (`eventing.knative.dev/v1`) and `channel` (`messaging.knative.dev/v1`) are predefined prefixes. Values in the configuration file can override these predefined prefixes.
+      - `group`: The APIGroup of Kubernetes resource.
+      - `version`: The version of Kubernetes resources.
+      - `resource`: The plural name of Kubernetes resources (for example:
+         `services`).
 
+* `channel-type-mappings` can be used to define aliases for custom channel types that can be used wherever a channel type is required (as in `kn channel create --type`). This configuration section defines an array of entries with the following fields:
```

Comment:
@abrennan89 this option is new and missing in the recent doc's PR. Please see also the full example above how this is used.

Reply:
Please open a doc issue for it an include the details 🙏🏻

Reply:
will do as soon as this is merged ...